app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, source, new = command.split(" ")
```

Comment:
Also before this validate that command.split() have 3 parameters
if len(command.split()) != 3:
    return
....